app/src/main/java/com/pajato/android/gamechat/common/BaseFragment.java

```diff
@@ -95,8 +97,9 @@ public BaseFragment() {}
         if (mLayout != null) return mLayout;
 
         // The layout does not exist.  Create and persist it, and initialize the fragment layout.
-        mLayout = inflater.inflate(getLayout(), container, false);
-        onInitialize();
+        mLayout = inflater.inflate(mLayoutId, container, false);
+        // All chat and game fragments will use the options menu.
```

Comment:
It would be better if you had one comment that applied to 5-8 lines of code with no intervening blank lines.  Consider this a code "paragraph".  Each such paragraph should be separated by a blank line.  In this way writing code mimics writing prose, a good thing, IMHO.